app/javascript/packs/application.scss

```diff
@@ -1,21 +1,12 @@
-/*
-Any CSS added to this file or imported from this file, e.g. `@import '../stylesheets/my-css.css'`,
-will be included in the "application" pack. Any CSS imported from application.js or as part of the
-application.js dependency graph, e.g. `import '../stylesheets/my-css.css'` will also be included
-in the "application" pack.
-
-To reference this file, add <%= stylesheet_pack_tag 'application' %> to the appropriate
-layout file, like app/views/layouts/application.html.erb
-*/
-
-@import 'bootstrap/scss/bootstrap';
-
-h1 {
-  text-decoration: underline;
-}
-
-body {
-  padding: 2em
-
-
-}
\ No newline at end of file
+/*
+Any CSS added to this file or imported from this file, e.g. `@import '../stylesheets/my-css.css'`,
+will be included in the "application" pack. Any CSS imported from application.js or as part of the
+application.js dependency graph, e.g. `import '../stylesheets/my-css.css'` will also be included
+in the "application" pack.
+
+To reference this file, add <%= stylesheet_pack_tag 'application' %> to the appropriate
+layout file, like app/views/layouts/application.html.erb
+*/
+
+@import 'bootstrap/scss/bootstrap';
+@import 'sakura'
```

Comment:
Should this get imported _before_ bootstrap so we don't accidentally override any styles?

Reply:
its fine to override bootstrap in the webpack bundle, because its only getting used for this one new page.  I'll remove the bootstrap import and comment on this here fact, just to make things more clear.